### ISSUE TITLE
Skins: Improve HTML skin container sizing

### ISSUE THREAD

**Issue by sampotts** [videojs/v10#808]
Currently we're using `display: contents` in the HTML skin examples and we shouldn't. There's some background in this slack thread:
https://mux.slack.com/archives/C096HT3S8F4/p1773027048905699
